webapp/packages/core-app/src/shared/NodesManager/NavNodeContextMenuService.ts

```diff
@@ -70,8 +71,12 @@ export class NavNodeContextMenuService extends Bootstrap {
           return !!node.features?.includes(ENodeFeature.canDelete);
         }
 
+        if (action === ACTION_OPEN) {
+          return node.nodeType !== RESOURCE_MANAGER_RESOURCE_NODE_TYPE
+            && node.nodeType !== RESOURCE_MANAGER_PROJECT_NODE_TYPE;
+        }
```

Comment:
we need to invert it to `is metadata object`
the better solution is to use `navNodeManagerService.navToNode` logic to add a handler for metadata objects and  use `is metadata object` for metadata handlers (placed in `plugin-object-viewer`)